### PR TITLE
feat: endless/tinycore/tails

### PR DESCRIPTION
**Pull request by curbengh**

### CODE REVIEW

File: peppermint/1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PRETTY_NAME="PeppermintOS"
+NAME="Peppermint"
+ID=peppermint
+VERSION_CODENAME="bookworm"
```
Comment (SteveLauC) [inline peppermint/1:4]: They have a `VERSION_CODENAME` but don't have a `VERSION_ID`

Comment (curbengh) [reply peppermint/1:4]: rename to `bookworm`?

Comment (SteveLauC) [reply peppermint/1:4]: I will enhance the naming convention to cover the edge cases of these 3 PRs

Comment (curbengh) [reply peppermint/1]: I've removed peppermint from this PR.

**Review by SteveLauC — APPROVED** [review]

File: peppermint/1 (hunk 1 above)
Comment (SteveLauC) [reply peppermint/1]: Thanks!

**Pull request merged**